### ISSUE TITLE
Can't install dependency pyethhash in amazon linux

### ISSUE THREAD

**Issue by sabotagebeats** [ApeWorX/ape#1367]
### Environment information
https://github.com/ethereum/ethash/issues/137

**Comment by unparalleled-js** [comment]
Is this a problem with the framework?

**Comment by fubuloubu** [comment]
No, I think it could also be resolved by having an image w/ gcc in it, or otherwise having a binary wheel for whatever that architecture is

**Comment by unparalleled-js** [comment]
Amazon has base images from Python Linux: https://github.com/aws/aws-lambda-base-images/blob/python3.9/Dockerfile.python3.9
